_includes/components/header--extended-mega-menu.html

```diff
@@ -0,0 +1,102 @@
+{% if header %}
+<header class="usa-header usa-header-extended" role="banner">
+
+  <div class="usa-navbar">
+    <button class="usa-menu-btn">Menu</button>
+    <div class="usa-logo" id="logo">
+      <em class="usa-logo-text">
+        <a href="{% if header.href %}{{ header.href }}{% else %}{{ site.baseurl }}/{% endif %}" title="Home">
+          {{ header.title | default: site.title }}
+        </a>
+      </em>
+    </div>
+  </div>
+
+  <nav role="navigation" class="usa-nav">
+    <div class="usa-nav-inner">
+      <button class="usa-nav-close">
+        <img src="{{ site.baseurl }}/assets/uswds/img/close.svg" alt="close">
+      </button>
+
+      {% assign _primary = header.primary.links %}
+      {% assign primary_links = site.data.navigation[_primary] | default: _primary %}
+      {% if primary_links %}
+      <ul class="usa-nav-primary usa-accordion">
+        {% for _section in primary_links %}
+        <li>
+          {% if _section.links %}
+            {% assign section_links = site.data.navigation[_section.links] | default: _section.links %}
+            {% assign _section_id = _section.id %}
+            {% unless _section_id %}{% assign _section_id = 'nav-' | append: forloop.index %}{% endunless %}
+          <button class="usa-accordion-button usa-nav-link" aria-expanded="false" aria-controls="{{ _section_id }}">
+            <span>{{ _section.text }}</span>
+          </button>
+          <ul id="{{ _section_id }}" class="usa-nav-submenu usa-megamenu usa-grid-full" aria-hidden="true">
+
+            {% for _link in section_links %}
+            {% capture modulo %}{{ forloop.index | modulo: 3 }}{% endcapture %}
+            {% if modulo == '1' %}
+              <div class="usa-megamenu-col">
```

Comment:
wrap every 3 links in a `usa-mengamenu-col` div

Reply:
Mind adding that as a [comment](https://shopify.github.io/liquid/tags/comment/)?